metrics/influxdb/influxdb.go

```diff
@@ -21,7 +21,7 @@ type reporter struct {
 	namespace string
 	tags      map[string]string
 
-	client *client.Client
+	client client.Client
 
 	cache map[string]int64
```

Comment:
I think this field is not used anymore

Reply:
What do you mean? It's used in both Ping and `send`
```
		case <-pingTicker.C:
			_, _, err := r.client.Ping(0)
```

```
	return r.client.Write(bps)
```